mail_tracking/static/src/js/mail_tracking.js

```diff
@@ -1,102 +1,105 @@
-            res_model: 'mail.tracking.email',
-            views: [[false, 'form']],
-            target: 'new',
-            res_id: tracking_email_id,
-        };
-        this.do_action(action);
-    },
-    bind_events: function () {
-        this.$el.on('click', '.o_mail_action_tracking_partner',
-                    this.on_tracking_partner_click);
-        this.$el.on('click', '.o_mail_action_tracking_status',
-                    this.on_tracking_status_click);
-    },
-    init: function (parent, options) {
-        this._super.apply(this, arguments);
-        this.action_manager = this.findAncestor(function(ancestor){
-            return ancestor instanceof ActionManager;
-        });
-    },
-    start: function () {
-        this._super();
-        this.bind_events();
-    },
-    render: function(messages, options) {
-        var self = this, render_super = this._super,
-            msgs = {},
-            msg_ids = [];
-        // Update trackings (async) each time we re-render thread
-        _.each(messages, function (message) {
-            msgs[message.id] = message;
-            msg_ids.push(message.id);
-        });
-        MessageModel.call('tracking_status', [msg_ids]).then(function (trackings) {
-            _.each(trackings, function (tracking, id) {
-                msgs[id].partner_trackings = tracking;
+    ChatThread.include({
+        on_tracking_partner_click: function (event) {
+            var partner_id = this.$el.find(event.currentTarget).data('partner');
+            var state = {
+                'model': 'res.partner',
+                'id': partner_id,
+                'title': _t("Tracking partner"),
+            };
+            event.preventDefault();
+            this.action_manager.do_push_state(state);
+            var action = {
+                type:'ir.actions.act_window',
+                view_type: 'form',
+                view_mode: 'form',
+                res_model: 'res.partner',
+                views: [[false, 'form']],
+                target: 'current',
+                res_id: partner_id,
+            };
+            this.do_action(action);
+        },
+        on_tracking_status_click: function (event) {
+            var tracking_email_id = $(event.currentTarget).data('tracking');
+            var state = {
+                'model': 'mail.tracking.email',
+                'id': tracking_email_id,
+                'title': _t("Message tracking"),
+            };
+            event.preventDefault();
+            this.action_manager.do_push_state(state);
+            var action = {
+                type:'ir.actions.act_window',
+                view_type: 'form',
+                view_mode: 'form',
+                res_model: 'mail.tracking.email',
+                views: [[false, 'form']],
+                target: 'new',
+                res_id: tracking_email_id,
+            };
+            this.do_action(action);
+        },
+        bind_events: function () {
```

Comment:
It's better to remove this method and add the events as a class property using [the extension method you can see here](https://github.com/odoo/odoo/blob/97fe1f05074a82715560cc825359663d65a39e41/addons/website/static/src/js/content/website_root.js#L38-L43).

This way we leverage Odoo framework to do the binding...